tests/e2e/util.go

```diff
@@ -4029,3 +4033,40 @@ func collectPodLogs(ctx context.Context, client clientset.Interface, namespace s
 		}
 	}
 }
+
+/*
+Type-2 testbed
```

Comment:
"Type-2 testbed" is not informative , Instead we can say that this method is for topology with two level labelling  testcases